docs/release.md

```diff
@@ -1,12 +1,15 @@
-**TL;DR**- If you simply need to learn how to make a release, read [this](#creating-releases) section.
+
+> **Note**
+> The procedure for cutting a release is automated and can be found [here](release-procedure.md).
 
 # Overview
-- Release process is automated by [hack/release.sh](https://github.com/kubevirt/kubevirt/blob/main/hack/release.sh) script
-- KubeVirt uses [semantic versioning](http://semver.org)
-- Primary artifact is the source tree in form of signed git tag
-- Binary artifacts are built using automation
-- The releases appear on a time based release schedule
-- Releases can be blocked using the ```release-blocker``` github comment
+- [Versioning](#versioning): KubeVirt uses [semantic versioning](http://semver.org)
+- [Content](#content): Primary artifact is the source tree in form of signed git tag. Binary artifacts are built for convenience by automation
+- [Platform Support](#platform-support): Every KubeVirt release is targeted towards one Kubernetes release
```

Comment:
Shouldn't this mention that two previous Kubernetes versions are also tested?

Reply:
yeah, If we're still planning to test compatiblity with older versions, we should be more explicit about this.
maybe something like... 
`Every KubeVirt release is targeted towards a specific Kubernetes release and is compatible with n-2 kubernetes releases`

Reply:
Fixed.